hw1/02-palindrome.js

```diff
@@ -1,3 +1,28 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
+const response = document.getElementById("response");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput() {
+  const inputNumber = parseInt(this.value);
```

Comment:
`parseInt` will silently ignore the fractional component of the input value.
For example, the function will say `121.345` is a palindromic number because it only looks at `121`.

I recommend using `parseFloat` instead and checking if there are any digits after the decimal point. This way, the function will be able to show the proper message for values like `121.345`.

Reply:
I didn't think about decimal points! I changed to parsfloat and it works great!